### PR TITLE
Use ppa plugin to access PPAs

### PR DESCRIPTION
**Pull request by AlanGriffiths**

### CODE REVIEW

File: snap/plugins/ppa.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        schema = super().schema()
+        schema['properties']['ppa'] = { 'type': 'string' }
+        schema['required'] = ['ppa']
+        return schema
+
+
+    def __init__(self, name, options, project):
+        super().__init__(name, options, project)
+        subprocess.run(["sudo", "apt", "--assume-yes", "install", "software-properties-common"])
+        for ppa in self.options.ppa.split(" "):
```
Comment (wmww) [inline snap/plugins/ppa.py]: Why are you parsing a space separated list of PPAs, instead of an actual YAML list?

**Review by wmww — CHANGES_REQUESTED** [review]

File: snap/plugins/ppa.py (hunk 1 above)
Comment (AlanGriffiths) [reply snap/plugins/ppa.py]: Mostly because the "normal" case is a single PPA.

Comment (wmww) [reply snap/plugins/ppa.py]: IMO it should either support a single PPA and be a string, or support multiple PPAs and be a list.

**Review by wmww — APPROVED** [review]
LGTM
bors r+

**Comment by AlanGriffiths** [comment]
> bors r+

@wmww It's a snap recipe, there's no bors.

**Pull request merged**